src/Native/pkg/runtime.native.System.Data.SqlClient.sni/runtime.native.System.Data.SqlClient.sni.pkgproj

```diff
@@ -4,13 +4,14 @@
   <PropertyGroup>
     <SkipPackageFileCheck>true</SkipPackageFileCheck>
     <SkipValidatePackage>true</SkipValidatePackage>
+    <PreReleaseLabel>servicing</PreReleaseLabel>
```

Comment:
@weshaggard I thought this package contains sni.dll that is built in the closed build and thus, the package builds there and not in the open. Is that not correct?

My proposal is to mark this as servicing in the TFS branch (which corresponds to master of CoreCLR and not release/1.1.0) to indicate the nature of the package.